otter/test/convergence/test_errors.py

```diff
@@ -49,7 +50,10 @@ def test_present_exceptions(self):
             CreateServerConfigurationError("Your server is wrong"):
                 'Server launch configuration is invalid: Your server is wrong',
             CreateServerOverQuoteError("You are over quota"):
-                'Servers cannot be created: You are over quota'
+                'Servers cannot be created: You are over quota',
+            NoSuchEndpoint(service_name="nova", region="ord"):
+                "Could not locate service nova in the service catalog. "
+                "Please check if your account is still active."
```

Comment:
Pro Tip: put a `,` at the end of this line to avoid it showing up in the next diff that modifies this dict (as the 'servers cannot be created' line shows up in this one because of the new comma...).